exporters/hk_create_employee.py

```diff
@@ -0,0 +1,40 @@
+import json
+import os
+import queue
+import threading
+import time
+import logging
+import codecs
+
+import click
+import requests
+from anytree import Node, PreOrderIter
+from datetime import datetime, timedelta
+
+
+from os2mo_helpers.mora_helpers import MoraHelper
+from os2mo_data_import import ImportHelper
+
+import holstebro_helpers as hh
+
+
+@click.command()
+@click.option('--dev', 'hostname', flag_value='https://os2mo-dev.holstebro.dk', help='https://os2mo-dev.holstebro.dk')
+@click.option('--test', 'hostname', flag_value='https://os2mo-test.holstebro.dk', help='https://os2mo-test.holstebro.dk')
+@click.option('--prod', 'hostname', flag_value='https://os2mo.holstebro.dk', help='https://os2mo.holstebro.dk')
+@click.option('--api_token', envvar='SAML_TOKEN', default=None, required=True, help='SAML API Token')
+@click.option('--employeeinfo', default=None, required=True, help='json formatted file with employee information')
+def create_employee(hostname, api_token, employeeinfo):
```

Comment:
Det ser ikke ud til at api_token benyttes?
```suggestion
@click.option('--employeeinfo', default=None, required=True, help='json formatted file with employee information')
def create_employee(hostname, employeeinfo):
```

Reply:
Næ. Jeg tror jeg fjerne create_employee helt fra Commit. Det var en midlertidlig løsning vi lavede pga. manglende integration til CPR service i MO UI.